partiql-parser/build.gradle.kts

```diff
@@ -90,6 +90,8 @@ tasks.processResources {
     from("src/main/antlr") {
         include("**/*.g4")
     }
+    // TODO remove in next major version release.
+    rename("PartiQLParser.g4", "PartiQL.g4")
```

Comment:
ACK. I'm going to keep it since we know that we plan on merging this to main to perform a release before V1. This should be removed once all of our deprecation notices are eventually deleted.